server/subscribe.go

```diff
@@ -269,6 +275,20 @@ func (p *Plugin) addChannelSubscription(newSubscription *ChannelSubscription) er
 	})
 }
 
+func (p *Plugin) isChannelSubscriptionNameUnique(channelId string, subscription *ChannelSubscription) error {
+	subs, err := p.getSubscriptionsForChannel(channelId)
+	if err != nil {
+		return err
+	}
+
+	for sub := range subs {
+		if subs[sub].Name == subscription.Name && subs[sub].Id != subscription.Id {
+			return errors.New("Subscription name, \"" + subs[sub].Name + "\", already exists. Please choose another name.")
```

Comment:
Nit: Change local var to `subID` and:
```suggestion
			return fmt.Errorf("Subscription name '%s' already exists. Please choose another name.", subs[subID].Name)
```